docs/_templates/layout.html

```diff
@@ -0,0 +1,8 @@
+{% extends "!layout.html" %}
+  {% block footer %} {{ super() }}
+
+  <style>
+         .wy-nav-content { max-width: 1000px; }
```

Comment:
what was the original `max-width` value?.. tbh I don't like the absolute pixels here, would very much like to use `em` unit (ref: https://www.w3schools.com/cssref/css_units.asp) to scale properly with viewer display's DPI.

Reply:
Original `max-width` value was 800px. Ok, set `max-width` to 65em.